src/components/tags/tags.js

```diff
@@ -85,20 +97,53 @@ export class TagsComponent {
 
   addTag(tag) {
     this.tagsList.innerHTML += tagItemTemplate({ tag: tag });
+    this.initMoreBtns();
   }
 
   handleEditClick() {
     console.log("handleEditClick");
   }
 
-  handleDeleteClick() {
-    console.log("handleDeleteClick");
+  handleDeleteClick(event) {
+    let moreButton = event.target.closest(".mdc-menu-anchor");
+    this.listItem = moreButton.closest(".tags__list-item");
+    let tagName = this.listItem.querySelector(".tags__list-item-name")
```

Comment:
to not search twice you can add to your <a> (which is the same as ".tags__list-item" element) data-name="<%= tag %>".